app/src/main/java/edu/uc/haflga/financialpal/AddTransaction.kt

```diff
@@ -4,7 +4,7 @@ import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import edu.uc.haflga.financialpal.ui.addtransaction.AddTransactionFragment
 
-class add_transaction : AppCompatActivity() {
+class AddTransaction : AppCompatActivity() {
```

Comment:
Changed name of class: https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules

Reply:
Good citation, and this does align with Kotlin naming conventions.

This duplicates a recommendation from an earlier code review on the same project.